### PR TITLE
add tags vsphere-ipi, 4.9 

### PR DESCRIPTION
**Pull request by dis016**
[OCPQE-5688](https://issues.redhat.com/browse/OCPQE-5688) added tags vsphere-ipi, 4.9 for prow jobs
/cc @jhou1 @JianLi-RH  @pruan-rht @liangxia

### CODE REVIEW

File: features/admin/quota.feature [hunk 1]
```diff
@@ -7,6 +7,7 @@ Feature: Quota related scenarios
   @gcp-ipi
   @4.9
   @aws-upi
+  @vsphere-ipi
   Scenario Outline: The quota usage should be incremented if meet the following requirement
```
Comment (jhou1) [inline features/admin/quota.feature:11]: For `Scenario Outline` please add the tag to the exact example header. See examples in https://github.com/openshift/verification-tests/pull/2332/files

Comment (liangxia) [reply features/admin/quota.feature:11]: It's fine there, as all the examples in this outline are able to run on all iaas.
But for scenario outlines which have examples on different iaas, it is a must to add tag to exact example header as suggest by @jhou1 stated above.

**Comment by liangxia** [comment]
/hold

To test the behavior of https://github.com/openshift/release/pull/22227
/lgtm

**Comment by liangxia** [comment]
Testing of openshift/release#22227 done.
/lgtm cancel

File: features/machine/machine.feature [hunk 2]
```diff
@@ -504,6 +509,7 @@ Feature: Machine features testing
   @admin
   @destructive
   @4.9
+  @vsphere-ipi
```
Comment (liangxia) [inline features/machine/machine.feature:512]: We may need to add the tag to specific `Examples` lines, as it contains scenario for several iaas there.

File: features/storage/cloud_provider.feature [hunk 3]
```diff
@@ -5,6 +5,7 @@ Feature: kubelet restart and node restart
   @destructive
   @inactive
   @4.9
+  @vsphere-ipi
```
Comment (liangxia) [inline features/storage/cloud_provider.feature:8]: The same here.

File: features/storage/node_operations.feature [hunk 4]
```diff
@@ -4,6 +4,7 @@ Feature: Node operations test scenarios
   @admin
   @destructive
   @4.9
+  @vsphere-ipi
```
Comment (liangxia) [inline features/storage/node_operations.feature:7]: Same here.

File: features/machine/machine.feature (hunk 2 above)
Comment (dis016) [reply features/machine/machine.feature]: updated the tags

**Pull request merged**